src/EbmlCrc32.cpp

```diff
@@ -199,7 +199,7 @@ void EbmlCrc32::AddElementCRC32(EbmlElement &ElementToCRC)
   //  Finalize();
 }
 
-bool EbmlCrc32::CheckElementCRC32(EbmlElement &ElementToCRC)
+bool EbmlCrc32::CheckElementCRC32(EbmlElement &ElementToCRC) const
```

Comment:
I believe this would break the ABI.

Reply:
Indeed, changing `const` on existing member functions does change the ABI. We have a lot of other ABI-breaking changes pending (in libMatroska, too, if I remember correctly), so we could bunch all of those together in a libEBML 2.0.0 release sometime soon.

Reply:
Should I remove them and place in a separate PR?